packages/flutter/lib/src/rendering/box.dart

```diff
@@ -2361,14 +2362,32 @@ abstract class RenderBox extends RenderObject {
       _cachedBaselines?.clear();
       _cachedIntrinsicDimensions?.clear();
       _cachedDryLayoutSizes?.clear();
-      if (parent is RenderObject) {
-        markParentNeedsLayout();
-        return;
-      }
+      _notifyParentIfDirty = true;
```

Comment:
Instead of introducing this field, could this method just return true/false to indicate whether caches were cleared?

Reply:
@goderbauer What's your opinion on command query separation ? returning a value would break it.

Reply:
> Instead of introducing this field, could this method just return true/false to indicate whether caches were cleared?

Done